src/javascripts/modules/pills.js

```diff
@@ -0,0 +1,40 @@
+export default class Pills {
+  constructor(el) {
+    this.el = el
+    this.setupListeners()
+  }
+
+  setupListeners() {
+    window.addEventListener('load', () => this.handleLoad())
```

Comment:
Instead of waiting for the entire page to load to kick off all the animations, you could also kick off each animation when each svg image is done loading. Look at the **image onload event** for that.